docs/source/releases.rst

```diff
@@ -0,0 +1,59 @@
+Release History
+###############
+
+
+v1.0.0-beta (2023-06-22)
```

Comment:
As far as picking a tag goes, I'd just say we should stick to the rules in https://peps.python.org/pep-0440/ or else the standard packaging tools will explode.

So we shouldn't actually append "beta" to the tag number for python installations... I'm not sure exactly which option in that pep is the right one here.

Reply:
v1.0.0 is fine, in my opinion

Reply:
We think this is v1.0.0 ready?  I guess I was following a general semver article not the pep one, good looking out.  

I can switch things to v1.0.0

Reply:
For the future record, I was apparently wrong about this and a version `v1.0.0-beta` does get normalized to `v1.0.0b0`

Reply:
For what it's worth, it's probably fine to not rely on that normalization if we can avoid it 🤷